doc/src/learn/cryptography/sui-intent-signing.md

```diff
@@ -46,7 +46,7 @@ The serialization of an `IntentMessage<T>` is the 3 bytes of the intent concaten
 
 ## User Signature
 
-To create a user signature, construct an intent message first, and create the signature over the BCS serialized value of the intent message (`intent || message`).
+To create a user signature, construct an intent message first, and create the signature over the Blake2b hash of the BCS serialized value of the intent message of the transaction data (`intent || message`).
```

Comment:
lets specifically mention the first 32 bytes. Realized many implementations out there output anything from 1..64 bytes.